### ISSUE TITLE
JSX Accessibility Error - Label Must Have Associated Control

### ISSUE THREAD

**Issue by saksham-gera** [ControlCore-Project/concore-editor#209]
The <label> tag in the OptionsModal.jsx file does not have accessible text, causing an ESLint error (jsx-a11y/label-has-associated-control).
